contracts/components/scanners/ScannerToNodeRunnerMigration.sol

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: UNLICENSED
+// See Forta Network License: https://github.com/forta-network/forta-contracts/blob/master/LICENSE.md
+
+pragma solidity ^0.8.9;
+
+import "../BaseComponentUpgradeable.sol";
+import "./ScannerRegistry.sol";
+import "./IScannerMigration.sol";
+import "../node_runners/NodeRunnerRegistry.sol";
+
+/**
+ * Migration of ScannerRegistry to NodeRunnerRegistry
+ */
+contract ScannerToNodeRunnerMigration is BaseComponentUpgradeable, IScannerMigration {
+    /** Contract version */
+    string public constant version = "0.1.0";
+    uint256 public constant NODE_RUNNER_NOT_MIGRATED = 0;
+    uint256 public constant MAX_SCANNERS = 100;
+
+    ScannerRegistry public scannerNodeRegistry;
+    NodeRunnerRegistry public nodeRunnerRegistry;
+    uint256 public migrationEndTime;
+
+    event SetScannerNodeRegistry(address registry);
+    event SetNodeRunnerRegistry(address registry);
+    event SetMigrationEndtime(uint256 migrationEndTime);
+    event MigrationExecuted(uint256 scannersMigrated, uint256 ignoredScanners, uint256 nodeRunnerId, bool mintedNodeRunner);
```

Comment:
Suggestions:

1. Renaming `ignoredScanners` to `scannersIgnored` for consistency with the first parameter.
2. Making `nodeRunnerId` indexed for ease of lookup.
4. Removing `mintedNodeRunner`. I feel less strongly about this, but it just seems unnecessary. What use case do you have in mind?

```suggestion
    event MigrationExecuted(uint256 scannersMigrated, uint256 scannersIgnored, uint256 indexed nodeRunnerId);
```

Reply:
1 and 2 I'm OK with.
Regarding 3, if someone has 1000 ScannerRegistry NFTs in the same chain, and only want 1 NodeRunnerRegistry NFT managing them, they will have to execute the migration method several times. The first 1 will mint a NodeRunnerRegistry and the following ones will need to get the new nodeRunnerId as input, hence `mintedNodeRunner` would be false

Reply:
I don't see exactly how `mintedNodeRunner` helps but if you're sure about it it's ok.

But I'm thinking now that we may want to add `address nodeRunner` as a parameter to make it easier to track the events. What do you think? Again I'm only roughly imagining what the migration transaction code will look like so I'm not entirely sure.

Reply:
I'm going to close this, since we are both not sure and at this point it's not as much a priority.